.github/ISSUE_TEMPLATE/---performance-problem.md

```diff
@@ -0,0 +1,58 @@
+---
+name: "\U0001F40C Performance Problem"
+about: Report an Issue related to performance problems when using Slimefun 4.
+title: ''
+labels: "\U0001F40C Performance Problem"
+assignees: ''
+
+---
+
+<!-- FILL IN THE FORM BELOW AND READ IT CAREFULLY -->
+<!-- Make sure to install Spark and follow the steps carefully to ensure that we can give you the best response possible. -->
+
+# :warning: Prerequisites
+1. Make sure you have read the wiki article on Server-Optimization: 
+   https://github.com/Slimefun/Slimefun4/wiki/Server-Optimizations
+2. Please install Spark: 
+   https://www.spigotmc.org/resources/spark.57242/)
+
+## :timer_clock: 1. Generic Timings Report (REQUIRED)
+<!-- Run `/timings on` and wait at least 5 minutes! -->
```

Comment:
Just wanted to comment on it 
Timings are on by default if you just let them run /timings paste it will get it from when the server started till they do the comment
In paper atleast

Reply:
Not in Spigot though afaik.